cmd/kola/options.go

```diff
@@ -33,8 +33,9 @@ var (
 	kolaPlatforms      = []string{"aws", "azure", "do", "esx", "gce", "openstack", "packet", "qemu", "qemu-unpriv"}
 	kolaDistros        = []string{"cl", "fcos", "rhcos"}
 	kolaDefaultImages  = map[string]string{
-		"amd64-usr": sdk.BuildRoot() + "/images/amd64-usr/latest/coreos_production_image.bin",
-		"arm64-usr": sdk.BuildRoot() + "/images/arm64-usr/latest/coreos_production_image.bin",
+		"amd64-usr":   sdk.BuildRoot() + "/images/amd64-usr/latest/coreos_production_image.bin",
+		"arm64-usr":   sdk.BuildRoot() + "/images/arm64-usr/latest/coreos_production_image.bin",
+		"ppc64le-usr": sdk.BuildRoot() + "/images/ppc64le-usr/latest/coreos_production_image.bin",
```

Comment:
This can be dropped, these default images are a legacy remnant of Container Linux.

Reply:
ack

Reply:
Looking at the extend of the change needed to dorp it and my rather limited knowledge of the code base. I will drop the ppc64le addition there as it shouldn't be need as per your comment.